src/generatePrefixesList.js

```diff
@@ -11,6 +11,14 @@ const clientId = "ntt-bgpalerter";
 const rpki = new RpkiValidator({clientId});
 import axiosEnrich from "./utils/axiosEnrich";
 
+import https from 'https';
+
+const httpsAgentOptions = {
+    keepAlive: (process.env.KEEP_ALIVE || 'true') === 'true',
+    maxSockets: parseInt(process.env.MAX_SOCKETS || '40', 10)
+}
+console.log(`Using HTTPS Agent Options: ${JSON.stringify(httpsAgentOptions)}`);
+axios.defaults.httpsAgent = new https.Agent(httpsAgentOptions);
```

Comment:
@forrert made getMultipleOrigins the error it received but I cannot tell if it is necessary. Do you think you should put that back in?

Reply:
@jsherin I don't see the change in master where Thomas made getMultipleOrigins the error it received. Can you give me a link

Reply:
I am referring to: https://github.com/appneta/BGPalerter/pull/4/files#diff-e943a4a381385ea088e7616d23007ce5652c485bf88a04e25bd1710826b92df6R57

Reply:
sounds good I added it the PR, it does not seem to introduce a problem, but I'm not sure it improves things